### DISCUSSION TITLE
关于v2ray-rules-dat geosite:category-games@cn的适配

### DISCUSSION THREAD

**Discussion by tossp** [IrineSistiana/mosdns#465] [Q&A] [answered]
https://github.com/Loyalsoldier/v2ray-rules-dat 中有一段描述

> 对于玩 Steam 国区游戏，想要直连的用户，可以设置类别 geosite:steam@cn 为直连，意为将 [steam](https://github.com/v2fly/domain-list-community/blob/master/data/steam) 列表内所有被标记了 @cn attribute 的规则（域名）设置为直连。同理，由于 [category-games](https://github.com/v2fly/domain-list-community/blob/master/data/category-games) 列表包含了 steam、ea、blizzard、epicgames 和 nintendo 等常见的游戏厂商。设置类别 geosite:category-games@cn 为直连，即可节省大量服务器流量。

这里`geosite:category-games@cn` 和 `query_matcher` 参数看起来有冲突，我尝试过用`./mosdns v2dat unpack-domain`提取提示没有该分类

**Comment by IrineSistiana** [comment, marked as answer]
query_matcher 没冲突。没问题。

只是 v2dat unpack-domain 解包的时候不支持用 `@attr` 过滤数据。

**Reply by tossp** [reply]
好的，谢谢
